spec/miq_ae_event_spec.rb

```diff
@@ -65,10 +65,31 @@
 
       context "with user owned VM" do
         let(:vm_owner) { user }
+        let(:miq_server) { EvmSpecHelper.local_miq_server(:is_master => true) }
+        let(:zone) { FactoryGirl.create(:zone) }
 
-        it "has tenant" do
+        it "raise_evm_event, deliver_queue, with zone" do
```

Comment:
@tinaafitz can we use a shared example for these 2 tests and just change the zones